libkbfs/keybase_service_base.go

```
@@ -681,3 +684,19 @@ func (k *KeybaseServiceBase) GetPublicCanonicalTLFNameAndID(
 
 	return res, nil
 }
+
+// EstablishMountDir asks the service for the current mount path
+func (k *KeybaseServiceBase) EstablishMountDir(ctx context.Context) (
+	string, error) {
+	dir, err := k.kbfsMountClient.GetCurrentMountDir(ctx)
+	if err != nil || dir == "" {
+		dirs, _ := k.kbfsMountClient.GetAllAvailableMountDirs(ctx)
```

Please don't ignore the error here.